server/events/vcs/common/common.go

```diff
@@ -30,7 +30,7 @@ func SplitComment(comment string, maxSize int, sepEnd string, sepStart string) [
 		if i > 0 {
 			portion = sepStart + portion
 		}
-		comments = append(comments, portion)
+		comments = append([]string{portion}, comments...)
```

Comment:
It would be nice to gate this feature behind a flag for people that do not want this behavior